### DISCUSSION TITLE
Project Migration to Webscope

### DISCUSSION THREAD

**Discussion by iamhectorsosa** [ekqt/screenshot#69] [General]
Hello, contributors:

Before anything, let me say that I acknowledge that there have been a lot of recent changes to our project. However, I'm very happy to see how much it has grown.

I'm excited to announce that the project's has been transferred to Webscope. I am part of this company and the folks at Webscope have shown interest in Screenshot and they want to support our project for further development. Therefore, I've agreed to hand the project over. I will continue to oversee and work with Screenshot as an OSS for the foreseeable future.

Please let me know if you have any questions, comments or thoughts.

I'm excited to see how Webscope support will shape the future of our project.

CC @kodbilenadam @alefDev-prog @mateusabelli.

Cheers

Hector

**Comment by kodbilenadam** [comment]
Can you tell us about the impacts of this change on the project?

Before when we raise an issue or a feature we can say it's useful or not and do it OSS way.

Is it gonna be like that or we gonna have a strict timeline and specific features to build which will be determined by the webscope?

**Reply by iamhectorsosa** [reply]
Great question! No impacts on any of these fronts. Issues and Features will be welcomed in the same way. I will continue reviewing all of these things and approve them as usual.

This change will bring us more resources in designers and developers joining the project. This will help shape the branding of the project and increase the quality of our code. It will still be kept as open source as we started it. ❤️

I understand that sometimes these changes bring limitations in development but I assure you that as long as I am heading the project, this will not be the case.

**Reply by kodbilenadam** [reply]
If that helps bring more people and keep the development cycle healthy I have no comment other  than welcoming webscope. 😊

**Reply by iamhectorsosa** [reply]
One of this project's main goals is to enable open source contribution! I believe strongly that Webscope will help all of us achieve that goal! A project from the community to the community! ❤️

**Reply by kodbilenadam** [reply]
Since the day one of starting contributing the project. The decisions you made and the directions you pointed are proved to me that this project is a really great way to get into open source.

Because of the way you handle the project / PR's and issues you have my trust developing this project further ❤️
